atlas-starter.py

```diff
@@ -0,0 +1,91 @@
+import pymongo
+
+# Replace the placeholder data with your Atlas connection string. Be sure it includes
+# a valid username and password! Note that in a production environment,
+# you should not store your password in plain-text here.
+
+client = pymongo.MongoClient(
+      "mongodb+srv://<username>:<password>@<cluster-name>/test?retryWrites=true&w=majority")
+
+# use a database named "my_database"
+
+db = client.my_database
+
+# use a collection named "recipes"
+
+my_collection = db["recipes"]
+
+recipe_documents = [{ "name": "elotes", "ingredients": ["corn", "mayonnaise", "cotija cheese", "sour cream", "lime"], "prep_time": 35 },
+                    { "name": "loco moco", "ingredients": ["ground beef", "butter", "onion", "egg", "bread bun", "mushrooms"], "prep_time": 54 },
+                    { "name": "patatas bravas", "ingredients": ["potato", "tomato", "olive oil", "onion", "garlic", "paprika"], "prep_time": 80 },
+                    { "name": "fried rice", "ingredients": ["rice", "soy sauce", "egg", "onion", "pea", "carrot", "sesame oil"], "prep_time": 40 }]
+
+# drop the collection in case it already exists
+
+my_collection.drop()  
+
+# INSERT DOCUMENTS
+#
+# You can insert individual documents using collection.insert_one().
+# In this example, we're going to create four documents and then 
+# insert them all with insert_many().
+
+my_collection.insert_many(recipe_documents)
+ 
+# FIND DOCUMENTS
+#
+# Now that we have data in Atlas, we can read it. To retrieve all of
+# the data in a collection, we call find() with an empty filter. 
+
+for doc in my_collection.find():
+  my_recipe = doc['name']
+  my_ingredient_count = len(doc['ingredients'])
+  my_prep_time = doc['prep_time']
+  print(f"{my_recipe} has {my_ingredient_count} ingredients and takes {my_prep_time} minutes to make.")
```

Comment:
Similar case here: I like the idea of demonstrating best practices like error handling in our examples.

Reply:
I added logic to handle the case where no records are found. Are you thinking to put the find() in a try block? There's really no good reason that operation should error at this point, and if it does the user would be better off seeing the stack trace than a generic error, but if all the other apps are using a try block I will as well.

Reply:
I waffled over this with java as well, but I think because these apps are meant as starting points/copy-pastable blocks it's probably a good idea to demonstrate best practices in all cases at the risk of some redundant error handling. I imagine it would be pretty easy for a user to delete the insert, change the collection, and try a find... only to hit an intimidating stack trace. It's certainly not our responsibility, but I figure we can at least reduce the chances with lots of try/catches.